sdk/src/loom/Graphics/Texture2D.ls

```diff
@@ -67,6 +84,17 @@ package loom.graphics
          * Fired when the texture is reloaded due to the backing asset changing.
          */
         public native var update:TextureInfoUpdateDelegate;
+
+        /**
+         * For asynchronous loads, this is fired when the texture has completed loading.
+         */
+        public native var asyncLoadComplete:TextureInfoAsyncLoadCompleteDelegate;
```

Comment:
I don't think so... the loading is so quick that I can't see where that would be useful tbh?  If it comes up as a request, it shouldn't be that hard to add, but I'd be in favor of not adding stuff for the sake of it, because that makes users feel like they should be/need to use it.

I vote to keep it in mind should any feedback come up from Roomlia, otherwise let's just KISS it as much as possible, no?